styles.css

```diff
@@ -422,7 +523,38 @@ textarea {
   color: #fff;
 }
 
+.btn-flex {
+  display: flex;
+}
+
+.btn-flex button:first-child {
+  margin-right: 12px;
+}
+
+.live_link,
+.source_link {
+  text-decoration: none !important;
+}
+
 @media (min-width: 768px) {
+  div.modal {
+    width: 65%;
+    height: 90vh;
+  }
+
+  div.modal_image {
+    height: 50%;
+  }
+
+  div.modal_flex {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  div.modal_flex div:first-child {
+    width: 60%;
+  }
+
   header {
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** As this detail was not mentioned by the previous reviewer, I really suggest you implement it before merging your PR. Kindly make sure you fix the overalaping `See Live` and `See Source` buttons styling in mobile version.
Here is how your design looks:
![Screenshot from 2023-01-04 23-41-31](https://user-images.githubusercontent.com/83042742/210656057-55db530a-8ef1-40a1-8538-f3d8240e2b18.png)